CHANGELOG.md

```diff
@@ -1,5 +1,219 @@
 # Changelog
 
+## [1.28.0] - ?
+
+The new release of rustup has been a long time in the making and comes with substantial changes.
+
+The following improvements might require changes to how you use rustup:
+
+- rustup will no longer automatically install the active toolchain if it is not installed. [pr#3985]
+
+  * To ensure its installation, run `rustup toolchain install` with no arguments.
+  * The following command installs the active toolchain both before and after this change:
+    ```sh
+    rustup show active-toolchain || rustup toolchain install
+    # Or, on older versions of PowerShell:
+    rustup show active-toolchain; if ($LASTEXITCODE -ne 0) { rustup toolchain install }
+    ```
+
+- Installing a host-incompatible toolchain via `rustup toolchain install` or `rustup default` will
+  now be rejected unless you explicitly add the `--force-non-host` flag. [pr#4028]
+
+rustup now officially supports the following host platforms:
+
+  - `aarch64-pc-windows-msvc` [pr#3840]
+  - `loongarch64-unknown-linux-musl` [pr#3921]
+
+We have additionally made the following internal changes:
+
+- The default download backend has been changed from reqwest with native-tls to reqwest with rustls. [pr#3798]
+  - `RUSTUP_USE_CURL` and `RUSTUP_USE_RUSTLS` can still be used to change the download backend
+    if the new backend causes issues.
```

Comment:
... and, if anything happens, please report to https://github.com/rust-lang/rustup/issues/3806.

Reply:
Added:

> If issues do happen, please [let us know](https://github.com/rust-lang/rustup/issues/3806).